dump/dbus_util.cpp

```diff
@@ -146,23 +149,19 @@ bool isHostRunning(sdbusplus::bus::bus& bus)
 {
     try
     {
-        constexpr auto hostStateObjPath = "/xyz/openbmc_project/state/host0";
-        auto retVal = readDBusProperty<DBusProgressValue_t>(
-            bus, "xyz.openbmc_project.State.Host", hostStateObjPath,
+        using PropertiesVariant =
+            sdbusplus::utility::dedup_variant_t<ProgressStages>;
+
+        auto retVal = readDBusProperty<PropertiesVariant>(
+            bus, "xyz.openbmc_project.State.Host",
+            "/xyz/openbmc_project/state/host0",
             "xyz.openbmc_project.State.Boot.Progress", "BootProgress");
-        const std::string* progPtr = std::get_if<std::string>(&retVal);
-        if (progPtr == nullptr)
+        const ProgressStages* progPtr = std::get_if<ProgressStages>(&retVal);
+        if (progPtr != nullptr)
```

Comment:
Do you want to return if it is not a nullptr?

Reply:
this method is called only during startup after that it will depend on property change request that comes to host_state_watch probably the reason this was not breaking but good we fixed